options/locale/locale_en-US.ini

```
@@ -1659,6 +1659,13 @@ pulls.switch_comparison_type = Switch comparison type
 pulls.switch_head_and_base = Switch head and base
 pulls.filter_branch = Filter branch
 pulls.no_results = No results found.
+pulls.show_all_commits = Show all commits
+pulls.show_changes_since_your_last_review = Show changes since your last review
+pulls.showing_only_single_commit = Showing only changes of commit %[1]s
+pulls.showing_specified_commit_range = Showing only changes between %[1]s..%[2]s
```

Hmm, I'm wondering if we should add the commit titles somewhere in these messages.
As a human, random numbers don't tell me too many things, and I have to remember which commit had that number.
It is certainly better than displaying nothing, but probably also not the most user-friendly approach.
What about something like
```suggestion
pulls.showing_only_single_commit = Showing only changes of commit %[1]s (%[2]s)
pulls.showing_specified_commit_range = Showing only changes between %[1]s..%[2]s (%[3]s..%[4]s)
```
Where we add the commit title of the corresponding commits (trimmed to 72 chars or something like that)

---

GitHub just shows "Showing changes from n commit(s)", which I think it also sufficient. The user already had selected the commits by title, so a short message is sufficient to not distort the UI with long messages.

---

i personally like the commit sha display - but if you want this to be changed, i can change it.

Do you want me to change it to only show the number of commits?

---

No, no, absolutely fine as it is.

---

but perhaps we should show how many commits this range consists of.

---

hm - not sure. this would extend the text and the provided information is (at least in my opinion) not that important.